bower.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "hawkular-ui-services",
   "description": "WIP:Angular Services for the Hawkular Console.",
-  "version": "0.8.16",
+  "version": "0.8.17",
```

Comment:
Another good thing about keeping the checkin (versioning) separate from the release is that the release can then be managed at a different level than the code checkin. And that decision can be made at a different time than code checkin.